data/yam/agama/auto/lib/base.libsonnet

```diff
@@ -1,7 +1,12 @@
+local stop_timeout() = {
+    'stopOnBootMenu' : true,
+};
+
+local timeout() = {
+    'timeout' : 15,
+};
+
```

Comment:
as I mentioned you don't need function, you can just use a block of json as the example below.